spec/datetimeformat.html

```diff
@@ -365,7 +365,7 @@ <h1>FormatDateTimePattern ( _dateTimeFormat_, _patternParts_, _x_, _rangeFormatO
           1. Else if _p_ is equal to *"fractionalSecondDigits"*, then
```

Comment:
This branch should assert that _dateTimeFormat_.[[FractionalSecondNumberFormat]] has the expected kind of value, since assignment to that slot is conditional.
```suggestion
          1. Else if _p_ is equal to *"fractionalSecondDigits"*, then
            1. Let _nf_ be _dateTimeFormat_.[[FractionalSecondNumberFormat]].
            1. Assert: Type(_nf_) is Object and _nf_ has an [[InitializedNumberFormat]] internal slot.
```